src/api/localResolvers/loan.js

```diff
@@ -20,7 +20,7 @@ function missingLoanField({
 	// Use optional chaining to see if nested loan properties exist
 	const value = propChain.reduce((val, chainLink) => val?.[chainLink] ?? null, loan);
 	if (value === null) {
-		logFormatter(`Loan resolver: Missing required field '${field}' for ${type}.${resolver}`, 'error', { loan });
+		logFormatter(`Loan resolver: Missing required field '${field}' for ${type}.${resolver}`, 'info', { loan });
```

Comment:
@emuvente This is my suggestion for now. The code safely handles this state and we know it can happen depending on the loan state.

Reply:
It may be that we can remove the log or possibly gate it to a lower env...

Reply:
We could change the check above to be for `undefined` instead of `null`. The intent of this log is to tell developers that they need to request those fields in their data fetching. But as you point out, sometimes these fields are missing, in which case the graphql api returns `null` which causes this log message to appear. If we check for `undefined` instead (and use `undefined` as the default value after the nullish operator), then this log should only occur when the fields aren't requested at all:
```js
const value = propChain.reduce((val, chainLink) => val?.[chainLink] ?? undefined, loan);
if (value === undefined) {
    ...
```

BTW when the values are `null` the `fundraisingPercent` resolver returns 0.

Reply:
I'm fine with that adjustment as well. I suppose we'd still want to return 0 in the failure case as that keeps this functioning safely.

Reply:
I went with this approach and changed the log to trigger on undefined instead